### PR TITLE
Pam 445 json schema editor

### PR DESCRIPTION
**Pull request by pamgoodrich**
Basic doc for:

1. Creating models and schemas.
2. JSON Schema Editor

**Rendered views**
Models:
https://meta.stoplight.io/docs/platform/branches/pam-445-json-schema-editor/3d00f2e0c2196-models-and-schemas

JSON Schema Editor:
https://meta.stoplight.io/docs/platform/branches/pam-445-json-schema-editor/zumkfdc16oypw-json-schema-editor

### CODE REVIEW

**Comment by heitortsergent** [comment]
This is outside of the files changed for this PR but just because it's linking to it, in the `Try It` page: https://github.com/stoplightio/platform-docs/blob/main/3.-design/try-it.md?plain=1#L20

I think we can make it clearer on the steps there for how to view the Try It option, something like:

```
To open Try It:

1. Edit a project.
2. Open an API description file
3. Select one of the API operations, and then select **Preview**.
4. Select the **Try It** tab. 
```

File: 3.-design/models.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+> ![Add model from context menu](https://stoplight.io/api/v1/projects/cHJqOjI/images/AI4lfAIuZPk)
+
+### The Model Editor
+
+The following example shows a data model for user information. This is a common use case for sharing data across APIs.
+
+![Model Editor](https://stoplight.io/api/v1/projects/cHJqOjI/images/OEu3MopWsDc)
+
+1. **Model name**
+2. **Model description**:  Basic Markdown can be used to format the description text.
```
Comment (heitortsergent) [inline 3.-design/models.md:42]: I know name and description are self-explanatory, but wondering if it's better to add something to it so it follows the same format as the rest of the list?

File: 3.-design/models.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+3. **Visibility**: Set the model as internal to hide it in public documentation and from guest users. See [API Visibility](../4.-documentation/set-internal-docs.md) for details.
+4. **JSON Schema Editor**: Use to build a schema object and examples. See [JSON Schema Editor](json-schema-editor.md) for details.
+5. **X-Extensions**: Add extensions to your model.
```
Comment (heitortsergent) [inline 3.-design/models.md:45]: Doesn't look like we have more documentation on what `x-extensions` are, maybe we can create a separate issue for it?

File: 3.-design/models.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+tags: []
+stoplight-id: 3d00f2e0c2196
+---
+
+# Models and Schemas
+
+Schemas define the request and response data types for an API. In Stoplight, the [JSON Schema Editor](json-schema-editor.md) provides an easy way to build OpenAPI Schema Objects.
```
Comment (heitortsergent) [inline 3.-design/models.md:8]: I'm not sure this is 100% right, saying schemas are for request and response data types?

I think Schemas are a way to represent a data structure that could represent an object, or an API request, or an API response for example.

File: 3.-design/models.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Models are a container for schemas, examples, and x-extensions. Models allow for reusability. Depending on how you create models, you can reuse them in a single API, in all APIs in a project, and in all APIs across a workspace (with design libraries). See [Shared Components](shared-components.md#shared-models) to learn how to share models once you've created them.
```
Comment (heitortsergent) [inline 3.-design/models.md:10]: Same here with Models being containers, I like what we have in the Shared Components, Models section: "Models allow you to describe these common structures (for example, a User object), and then reference the object from endpoint definitions, or even from other models."

Maybe mixing the two together:

"Models allow you to describe common data structures (for example, a User object), and can contain schemas, examples, and x-extensions", or something like that.

**Review by heitortsergent — APPROVED** [review]

File: 3.-design/api-design-editor.md [hunk 5]
```diff
@@ -76,9 +76,9 @@ You can customize the published [project sidebar](https://meta.stoplight.io/docs
 Use the main editor panel to design APIs, write supporting documentation, and update style guides. 
 
 Choose different views:
- - **Form**: Offers an intuitive editor for APIs, endpoints, and models. 
+ - **Form**: Offers an intuitive editor for APIs, endpoints, and models. Use the [JSON Schema Editor](json-schema-editor.md) to design data mode for requests and responses.
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 3.-design/api-design-editor.md:79]: modes should have been models. Fixed.

File: 3.-design/design-overview.md [hunk 6]
```diff
@@ -22,6 +22,17 @@ Once you have your API described in Stoplight Studio, you can:
...
+There are a few ways to get started designing your API with Stoplight Studio:
+
+- [Design-First Principles](https://meta.stoplight.io/docs/api-best-practices/ZG9jOjM2NTM5Nzc2-design-first-principles)
+- [Models and Schemas](models.md)
+- [JSON Schema Editor](json-schema-editor.md)
+- [Shared Components](shared-components.md)
+- [Validation and Linting](validation-and-linting.md)
+- [Understand the Directory Structure](../7.-projects/working-with-files.md#directory-structure)
+
 ### Studio Web vs. Studio Desktop
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 3.-design/design-overview.md:36]: Good catch! Fixed.

File: 3.-design/json-schema-editor.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+While each method can be used individually, you'll likely find yourself using a combination of both methods while [creating schemas and models](models.md). 
+
+Here's a simple example of a schema in the **Form Editor**. The schema contains a single object with:
+
+* `feature-name` string field that's required
+* `feature-flag` boolean field that has `false` set as the default value
+
+![Simple JSON Schema Form Editor Example](https://stoplight.io/api/v1/projects/cHJqOjI/images/3gRshcZa320)
+
+JSON is automatically generated from what you configure in the **Form Editor**. Here's is the **Code** view for the same example:
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 3.-design/json-schema-editor.md:30]: Good catch. I changed the wording to this:

Depending on the format of your API, JSON or YAML are automatically generated from what you configure in the **Form Editor**. Here's the generated JSON in the **Code** view for the same example:

File: 3.-design/models.md (hunk 1 above)
Comment (pamgoodrich) [reply 3.-design/models.md:42]: Yeah... I went back and forth. Added some wording.

File: 3.-design/models.md (hunk 3 above)
Comment (pamgoodrich) [reply 3.-design/models.md:8]: I changed it to this: Schemas define data structures for an API.

File: 3.-design/models.md (hunk 4 above)
Comment (pamgoodrich) [reply 3.-design/models.md:10]: Yeah, I'll do that. This was tough, honestly. I'll use this:

Models allow you to describe common data structures (for example, a User object), and can contain schemas, examples, and x-extensions.

File: 3.-design/models.md (hunk 2 above)
Comment (pamgoodrich) [reply 3.-design/models.md:45]: I have an issue for it for the next round of updates.

**Pull request merged**